djangocms_text_ckeditor/static/djangocms_text_ckeditor/ckeditor_plugins/cmsplugins/plugin.js

```diff
@@ -13,7 +13,7 @@
      * @returns {String} div|span
      */
     function getFakePluginElement(pluginMarkup) {
-        var innerTags = (pluginMarkup.match(/<([\S]*?)\s[\s\S]*?>/) || [0, false]).splice(1);
+        var innerTags = (pluginMarkup.match(/<\s*([^>\s]+)[\s\S]*?>/) || [0, false]).splice(1);
```

Comment:
wouldn't it be the same as `/\s*(\S+?)[\s\S]*?>/`? although it's not much of a difference i'd prefer a smaller construct